maas/client/viscera/machines.py

```diff
@@ -35,7 +35,7 @@ def read(cls):
         return cls(cls)
 
     def allocate(
-            cls, *, hostname: str=None, architecture: str=None,
+            cls, *, name: str=None, architecture: str=None,
```

Comment:
I wonder if we should stick with _hostname_ here. What's better: a clearer parameter name, or consistency with the back-end?